.github/workflows/create-releases.yml

```
@@ -0,0 +1,34 @@
+name: Create releases and tags
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+  
+jobs:
+  unreleased-prs-metadata:
+    name: Get list of pending release pull requests
+    permissions:
+      pull-requests: read
+      contents: read
+    uses: equinor/radix-reusable-workflows/.github/workflows/template-unreleased-pr-metadata.yml@v1.0.1
```

nitpick, add a newline to seperate the jobs :) 

```suggestion
    uses: equinor/radix-reusable-workflows/.github/workflows/template-unreleased-pr-metadata.yml@v1.0.1
    
```